test/dynamo/test_modules.py

```diff
@@ -683,6 +692,17 @@ def forward(self, x):
         return x * self.scale
 
 
+class RegNet(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        model = torchvision_models.regnet_y_400mf()
```

Comment:
I purposely use this complicated module architecture to test if we have a good coverage.

Reply:
Running all of resnet is pretty slow, I would prefer a minimal repro instead. To test a full model, we should see why our benchmark suite did not catch it

Reply:
Make sense, I updated with a simplified version. Thanks.